testing/trino-tests/src/test/java/io/trino/function/TestingConnectorTableFunction.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.function;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.connector.ConnectorTransactionHandle;
+import io.trino.spi.ptf.AbstractConnectorTableFunction;
+import io.trino.spi.ptf.Argument;
+import io.trino.spi.ptf.ConnectorTableFunction;
+import io.trino.spi.ptf.Descriptor;
+import io.trino.spi.ptf.TableFunctionAnalysis;
+
+import javax.inject.Provider;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static io.trino.spi.ptf.ReturnTypeSpecification.GenericTable.GENERIC_TABLE;
+import static io.trino.spi.type.BigintType.BIGINT;
+
+public class TestingConnectorTableFunction
```

Comment:
We've got a class `TestingTableFunctions`. Currently it has one TF implementation. You can use te existing implementation or add your implementation there.